lib/querystring.js

```diff
@@ -15,6 +15,7 @@ const QueryString = module.exports = {
   decode: parse
 };
 const Buffer = require('buffer').Buffer;
+const hexTable = require('internal/querystring').hexTable;
```

Comment:
I initially followed the style of existing imports in the file (see `Buffer` above), but since I'm importing multiple items from this module now I started using destructuring.